Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -134,6 +134,11 @@
     pointSize : "5"
 });
 
+addStyle('Clamp and Mix', {
+    color : "clamp(color(), 0.0, 100.0)",
+    pointSize : "mix(0.0, 2.0, 0.5) * 5"
+});
```

Comment:
Ah my mistake here, the shader doesn't compile because `color: clamp(${temperature}, 0.1, 0.2)` doesn't result in a color, but `"color() * clamp(${temperature}, 0.1, 0.2)"` should do the trick.

Can you also include `temperature` in the `pointSize` calculation.